src/main/java/thePackmaster/powers/monsterhunterpack/HuntersMark.java

```diff
@@ -22,7 +22,7 @@ public class HuntersMark extends AbstractPackmasterPower {
     public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
 
     public HuntersMark(AbstractCreature owner, int amount) {
-        super(POWER_ID, LOC_NAME, PowerType.DEBUFF, false, owner, amount);
+        super(POWER_ID, LOC_NAME, PowerType.DEBUFF, true, owner, amount);
```

Comment:
isTurnBased only changes the color of the number the power renders, it doesn't actually make the power turn based, you'll have to implement that logic yourself